services/nginz/integration-test/conf/nginz/nginx.conf

```diff
@@ -244,6 +244,11 @@ http {
       proxy_pass http://brig;
     }
 
+    location ~* /teams/([^/]+)/services {
```

Comment:
it's not nice that we have so many config files that should in fact all be the same :(